doc/text/news.md

```diff
@@ -1,13 +1,13 @@
 # News
 
-## 3.4.4 - 2021-06-04 {#version-3-4-4}
```

Comment:
I want to keep the `#version-3-4-4` anchor for referencing the specified version.

Reply:
`#3-4-4-2021-06-04` is the new anchor. `version-3-4-4` is the desirable for you? I can understand it👍 (Bu t I don't know hot to realize it...)

Reply:
Yes.
But it's not a strong requirement.